src/test/java/org.spongepowered.mod/test/integration/InventoryTest.java

```diff
@@ -0,0 +1,56 @@
+package org.spongepowered.mod.test.integration;
```

Comment:
License

Reply:
The file path is wrong too. It needs to be corrected.

Reply:
Fixed